### ISSUE TITLE
backgroundimages urls are tryed to get parsed and result in error

### ISSUE THREAD

**Issue by X-Tender** [arccoza/postcss-if-media#3]
`background-image: url("https://placem.at/places?h=375&w=375");`

Result in:

```
postcss-if-media: /PATHTOSCSS/file.scss:16:3: Invalid inline query. background-image: url("https://placem.at/places?h=375&w=375")
```

It's because of the `?`

**Comment by arccoza** [comment]
Thanks, I'll look into it.

**Comment by arccoza** [comment]
Fixed in 8b262ef, and available on npm as v1.0.2.
